packages/owlbot-bootstrapper/cloudbuild-owlbot-bootstrapper.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Run by a CLI command manually (by a human), or when googleapis/googleapis API Index changes
+# In order to run containers, so that languages can have their own isolated workflow to generate
+# their language-specific repositories, we need to run these containers in Cloud Build. This file
+# will trigger the pre- and post- processing steps of a language-specific container, as well as running
+# the language-specific container itself.
+
+steps:
+  # Clones the target repository and opens a branch for mono-repos, for split-repos creates a repository
+  - name: ${_CONTAINER}
+    env:
+    - 'LANGUAGE=$_LANGUAGE'
+    - 'PROJECT_ID=$PROJECT_ID'
+    - 'API_ID=$_API_ID'
+    - 'REPO_TO_CLONE=$_REPO_TO_CLONE'
+    - 'IS_PRE_PROCESS=$_IS_PRE_PROCESS'
```

Comment:
The reason I chose to use an env variable is because we're triggering a cloud build script with the command once, and then the cloud build script just runs the containers in sequence. In order to determine whether it should be a 'pre' or 'post' process, the cloud build file itself changes the env variable [here](https://github.com/googleapis/repo-automation-bots/blob/bc193dc56ce052471f663b5ae3cb3430b70c39db/packages/owlbot-bootstrapper/cloudbuild-owlbot-bootstrapper.yaml#L50).

Since we're only using the command-line call once, how would we alternate between the pre- and post- steps using just the `command` argument?

Reply:
Talked it over, implemented!